src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -110,7 +110,7 @@ internal class ContributionProvider
         {
```

Comment:
Add or update existing spec test to verify `.publish.json` contains correct `source_url` for different content types and files in dependency repos.

Reply:
sure, I am still working on spec tests